src/openforms/variables/models.py

```diff
@@ -0,0 +1,86 @@
+from django.core.exceptions import ValidationError
+from django.db import models
+from django.utils.translation import gettext_lazy as _
+
+import jq
+from json_logic import jsonLogic
+
+from .constants import DataMappingTypes, ServiceFetchMethods
+
+
+class ServiceFetchConfiguration(models.Model):
+    service = models.ForeignKey(
+        "zgw_consumers.Service",
+        on_delete=models.PROTECT,
+    )
+    path = models.CharField(
+        _("path"),
+        max_length=250,
+        blank=True,  # allow "" for the root, but not None
+        default="",
+        help_text=_("path relative to the Service API root"),
+    )
+    method = models.CharField(
+        _("HTTP method"),
+        max_length=4,
+        choices=ServiceFetchMethods.choices,
+        default="GET",
+        help_text=_("POST is allowed, but should not be used to mutate data"),
+    )
+    headers = models.JSONField(
+        _("HTTP request headers"),
+        blank=True,
+        null=True,
+        help_text=_(
+            "Additions and overrides for the HTTP request headers as defined in the Service."
+        ),
+    )
```

Comment:
probably good to add a validator here so that we don't get illegal header names/values. E.g. `{"foo bar": 30.0}` does not make any sense